composer.json

```diff
@@ -23,16 +23,17 @@
         "docs": "http://docs.portphp.org"
     },
     "require": {
-        "php": ">=5.4.0"
+        "php": ">=5.5.0",
```

Comment:
We should require at least php 5.6, because portphp/portphp requires at least php 5.6.

Reply:
Fixed